modules/installation-configuration-parameters.adoc

```diff
@@ -524,9 +510,9 @@ networking:
 |`networking.machineNetwork.cidr`
 |
 Required if you use `networking.machineNetwork`. An IP address block. The default value is `10.0.0.0/16` for all platforms other than libvirt and {ibmpowerProductName} Virtual Server. For libvirt, the default value is `192.168.126.0/24`. For {ibmpowerProductName} Virtual Server, the default value is `192.168.0.0/24`.
-ifdef::ibm-cloud-vpc[]
-The CIDR must contain the subnets defined in `platform.ibmcloud.controlPlaneSubnets` and `platform.ibmcloud.computeSubnets`.
-endif::ibm-cloud-vpc[]
+ifdef::ibm-cloud[]
+If you are deploying the cluster to an existing Virtual Private Cloud (VPC), the CIDR must contain the subnets defined in `platform.ibmcloud.controlPlaneSubnets` and `platform.ibmcloud.computeSubnets`.
+endif::ibm-cloud[]
```

Comment:
@gpei Thank you for looking at this. Very small change here. I removed the `ibm-cloud-vpc` attribute to accommodate this refactor. If you are curious about this effort, you can find more details here [1]

[1] https://issues.redhat.com/browse/OSDOCS-6493

Reply:
@mjpytlak thanks, this change looks good to me